plugins/catalog-backend/src/catalog/DatabaseLocationsCatalog.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { DatabaseLocationsCatalog } from './DatabaseLocationsCatalog';
+jest.mock('../ingestion/LocationReaders');
+
+import knex from 'knex';
+import path from 'path';
+
+import { Database } from '../database';
+import { getVoidLogger } from '../../../../packages/backend-common/src/logging/voidLogger';
+import { ReaderOutput } from '../ingestion/types';
+
+describe('DatabaseLocationsCatalog', () => {
+  const database = knex({
+    client: 'sqlite3',
+    connection: ':memory:',
+    useNullAsDefault: true,
+  });
+  database.client.pool.on('createSuccess', (_eventId: any, resource: any) => {
+    resource.run('PRAGMA foreign_keys = ON', () => {});
+  });
```

Comment:
let's move this into a `testutil.ts` file in the package src root in a separate pr sometime ...